File: src/1295-Find-Numbers-with-Even-Number-of-Digits/1295.js
var findNumbers = function(nums) {
    let res = 0;
    for (let n of nums) {
        let x = 0;
        while (n) {
            n = Math.floor(n / 10);
            x++;
        }
        if (x % 2 == 0) res++;
    }
    return res;
};

File: src/1295-Find-Numbers-with-Even-Number-of-Digits/1295.go
func findNumbers(nums []int) int {
    res := 0
    for _, v := range nums {
        x := 0
        for v != 0 {
            v /= 10
            x++
        }
        if x % 2 == 0 {
            res++
        }
    }
    return res
}